### ISSUE TITLE
[BUG] Food "Items" missing options in V12

### ISSUE THREAD

**Issue by BenNelsonNeb** [roth-michael/FoundryVTT-RestRecovery#269]
**Describe the bug**
For Rest and Recovery for 5e, items no longer have the options to select food or water or both. Or if it counts as 1 or whole day. 

**Additional context**

**Comment by roth-michael** [comment]
That certainly sounds like an issue. I can't seem to recreate it, though. Is the item set as a consumable, and the consumable type set to "Food"?

**Comment by BenNelsonNeb** [comment]
> That certainly sounds like an issue. I can't seem to recreate it, though. Is the item set as a consumable, and the consumable type set to "Food"?

Yes. I thought it might be a V12 thing, but maybe its another module causing conflicts. See attachment.
![Screenshot 2024-08-21 140317](https://github.com/user-attachments/assets/3c9886be-151a-4136-82c1-d9480d6e1c44)

**Comment by roth-michael** [comment]
Yeah, try disabling all modules besides Rest Recovery and let me know if the issue persists. If not, and you figure out what module is causing the issue, maybe I can address it from my side.

**Comment by BenNelsonNeb** [comment]
> Yeah, try disabling all modules besides Rest Recovery and let me know if the issue persists. If not, and you figure out what module is causing the issue, maybe I can address it from my side.

Just disable all but Rest and Recovery 5e. Same thing. I tried in both Firefox and Edge. To make sure....there should be additional fields under "Consumable Properties". Foundry V12 and D&D 5e v3.3.1

**Comment by BenNelsonNeb** [comment]
Brought up the console log, no errors when looking at or using an existing item that was working before. Also, no errors when attempting to create a new consumable > food.  I still don't see any additional options. Not sure if there is a certain option to trigger the fields?  I have food/water enabled in the module. 
![Screenshot 2024-08-21 143744](https://github.com/user-attachments/assets/b537968a-1476-4895-b4e5-2a0fdf393550)

**Comment by roth-michael** [comment]
Shouldn't be any specific option to do it, no. For context, the following is what I see:
![image](https://github.com/user-attachments/assets/86e09391-d67b-444f-a01e-1c7dd9dda049)

Does the issue persist if you create a new world?

**Comment by BenNelsonNeb** [comment]
Created a new world and only had RR 5e active.  I still don't see those options. Let me completely remove the module and reinstall it.

**Comment by roth-michael** [comment]
What browser are you using? Wondering if any extensions could be somehow getting in the way.

**Comment by BenNelsonNeb** [comment]
Found the trick. In the new world I thought those option would just appear on the items, but you do have to in to the settings and enable the option for "Water and Food Automation".  Once I did that I saw it in my new world. 

I went back to my main world that was still having the issue and turned off the "Water and Food Automation" saved and turned it back on.  Those options came back with a toggle. 

My guess is that my settings stored in my main world for RR 5e were corrupt, and toggling an option rewrote the setting correctly.

Issues can be resolved.  Thanks for your help.
